routing/turns_tts_text.cpp

```diff
@@ -62,11 +67,70 @@ string GetTtsText::GetTurnNotification(Notification const & notification) const
   if (notification.m_useThenInsteadOfDistance && notification.m_turnDir == CarDirection::None)
     return string();
 
-  string const dirStr = GetTextById(GetDirectionTextId(notification));
   if (dirStr.empty())
     return string();
 
   string const distStr = GetTextById(GetDistanceTextId(notification));
+
+  if (notification.m_distanceUnits > 0 && !notification.m_nextStreet.empty()) {
+    // We're going to pronounce the street name.
+
+    // We have to use wstring in order to operate on Unicode text
```

Comment:
No, we don't. See base/string_utils.hpp, `UniString`, `UniChar` and other helper methods.
codecvt is not implemented well on all systems at the moment and may not work properly.